ci/benchmark/build.sh

```diff
@@ -77,11 +77,16 @@ conda install "rmm=$MINOR_VERSION.*" "cudatoolkit=$CUDA_REL" \
 # conda remove -f rapids-build-env rapids-notebook-env
 # conda install "your-pkg=1.0.0"
 
-# Install the master version of dask, distributed, and streamz
-logger "pip install git+https://github.com/dask/distributed.git@$DASK_DISTRIBUTED_GIT_TAG --upgrade --no-deps"
-pip install "git+https://github.com/dask/distributed.git@$DASK_DISTRIBUTED_GIT_TAG" --upgrade --no-deps
-logger "pip install git+https://github.com/dask/dask.git@$DASK_DISTRIBUTED_GIT_TAG --upgrade --no-deps"
-pip install "git+https://github.com/dask/dask.git@$DASK_DISTRIBUTED_GIT_TAG" --upgrade --no-deps
+# Install the conda-forge or nightly version of dask and distributed
+if [[ "${INSTALL_DASK_MAIN}" == 1 ]]; then
+    gpuci_logger "gpuci_mamba_retry update dask"
+    gpuci_mamba_retry update dask
+else
+    gpuci_logger "gpuci_mamba_retry install -c conda-forge dask>=2021.11.1"
+    gpuci_mamba_retry install -c conda-forge dask
```

Comment:
Have we verified that these commands successfully install conda-forge packages over the nightlies? I would think we'd potentially run into the same issues we had before since this is considered a downgrade

Reply:
Ideally the image will have `dask` channel removed in this case but have made a test commit to check this.

Reply:
```suggestion
    gpuci_logger "gpuci_mamba_retry install conda-forge::dask>=2021.11.1 --force-reinstall"
    gpuci_mamba_retry install conda-forge::dask>=2021.11.1 --force-reinstall
```